src/TouristInfoCards.js

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+import "./css/TouristInfoCards.css";
+import londonPic from "./images/ london-web.jpg";
+import glasgowPic from "./images/glasgow-web.jpg";
+import manchesterPic from "./images/manchester_web.jpg";
+
+const TouristInfoCard = props => {
```

Comment:
It's really great that decided to split out the `TouristInfoCard` component! It makes your code much more flexible :)

But remember that we always put components in separate files so that we don't get confused. Could you move this to a separate file?